spk/sickchill/src/service-setup.sh

```diff
@@ -0,0 +1,52 @@
+PYTHON_DIR="/usr/local/python3"
+GIT_DIR="/usr/local/git"
+PATH="${SYNOPKG_PKGDEST}/bin:${SYNOPKG_PKGDEST}/env/bin:${PYTHON_DIR}/bin:${GIT_DIR}/bin:${PATH}"
+HOME="${SYNOPKG_PKGDEST}/var"
+VIRTUALENV="${PYTHON_DIR}/bin/virtualenv"
+PYTHON="${SYNOPKG_PKGDEST}/env/bin/python"
+SC_INSTALL_DIR="${SYNOPKG_PKGDEST}/share/SickChill"
+SC_BINARY="${SC_INSTALL_DIR}/SickChill.py"
+SC_DATA_DIR="${SYNOPKG_PKGDEST}/var/data"
+SC_CFG_FILE="${SC_DATA_DIR}/config.ini"
+SC_CONFIG_GIT_BRANCH=master
+SC_CONFIG_GIT_COMMIT_HASH=2190ef19c06b27911c01e04d5310220bcd6ac929
+SC_CONFIG_GIT_COMMIT_BRANCH=v2020.09.28-2
```

Comment:
To ease update task, these values should come from cross/sickchill/Makefile itself.
I propose that makefile generate a small file with these variables that `set_config` read at runtime to create/update configuration file.